client/landing/stepper/declarative-flow/internals/steps-repository/goals/select-card.tsx

```diff
@@ -0,0 +1,43 @@
+import { CheckboxControl } from '@wordpress/components';
+import { useInstanceId } from '@wordpress/compose';
+import classNames from 'classnames';
+import React from 'react';
+
+type SelectCardProps = {
+	children: React.ReactNode;
+	className?: string;
+	onChange: ( checked: boolean, value: string ) => void;
+	selected: boolean;
+	value: string;
+};
```

Comment:
Maybe we could consider moving the `SelectCardProps` type definition out to `client/landing/stepper/declarative-flow/internals/steps-repository/goals/types.ts`.

Reply:
Good question, this is a decision we would have to apply for all the next components. We could let only the data-related types or keep both UI and data types in the `types.ts`. I tend to prefer the first approach, keeping the UI types close to the components since they are not used by other files. What do you think?

Reply:
Yeah, I think that makes sense to leave the types definition as it is now. 👍🏼